tests/ut-ncnnapi.cc

```diff
@@ -85,4 +85,20 @@ TEST(ncnnapi, service_predict)
   cl1 = jd["body"]["predictions"][0]["classes"][0]["cat"].GetString();
   ASSERT_TRUE(jd["body"]["predictions"][0]["classes"][0]["prob"].GetDouble()
               > 0.4);
+
+  // predict with scale, wrong value, for testing only
+  jpredictstr
+      = "{\"service\":\"imgserv\",\"parameters\":{\"input\":{\"height\":300,"
+        "\"width\":300,\"scale\":0.0039},"
+        "\"output\":{\"bbox\":true}},\"data\":[\""
+        + incept_repo + "face.jpg\"]}";
+  joutstr = japi.jrender(japi.service_predict(jpredictstr));
+  std::cout << "joutstr=" << joutstr << std::endl;
+  jd.Parse<rapidjson::kParseNanAndInfFlag>(joutstr.c_str());
+  ASSERT_TRUE(!jd.HasParseError());
+  ASSERT_EQ(200, jd["status"]["code"]);
+  ASSERT_TRUE(jd["body"]["predictions"].IsArray());
+  cl1 = jd["body"]["predictions"][0]["classes"][0]["cat"].GetString();
+  ASSERT_TRUE(jd["body"]["predictions"][0]["classes"][0]["prob"].GetDouble()
+              > 0.4);
```

Comment:
This test checks that there are no errors when providing scale, but offer no guarantee that the scale is correctly applied

Reply:
Yes, we don't have models like this yet, at least it goes through the code, that uses the same NCNN mat function as `mean` and `std`.

Reply:
I can try to export a pytorch model to ncnn... Maybe in the future :)